pylint/checkers/refactoring.py

```diff
@@ -658,12 +663,39 @@ def _find_lower_upper_bounds(comparison_node, lower_bounds, upper_bounds):
             self.add_message('chained-comparison',
                              node=node)
 
+    def _check_possible_any_all_usage(self, boolop_node):
+        if boolop_node.op not in ('and', 'or'):
+            return
+
+        def _get_operands(bool_op, operator):
+            values = []
+            if bool_op.op != operator:
+                return values
+
+            for value in bool_op.values:
+                if isinstance(value, astroid.Name):
+                    values.append(value.name)
+                elif isinstance(value, astroid.Call):
+                    values.append(value.func.as_string())
+            return values
+
+        possible_any_operands = _get_operands(boolop_node, 'or')
+        # any/all makes sense only if there more than element in sequence
+        if len(possible_any_operands) > 1:
```

Comment:
This block could be refactored to a `for` loop, where we verify each operator individually.

Reply:
yup.